### PR TITLE
Change Readme

### PR DESCRIPTION
**Pull request by T-R-I-X**
Reformat readme.md to be more concise and cover all requirements that LegionCore specifically requires. Also list the discord community server I am hosting WIP.

Also I want to start on a github wiki so it takes the projects reliance on the TrinityCore wiki and removes it.

### CODE REVIEW

**Comment by dufernst** [comment]
Thanks for the merge request! I left some comments :)

File: README.md [hunk 1]
```diff
@@ -1,59 +1,49 @@
+![image](https://user-images.githubusercontent.com/24784631/216797451-f3d58b02-c900-4d37-9a3c-03ce48ffc411.png)
```
Comment (dufernst) [inline README.md:1]: I wouldn't mind not having the logo, not adding much :D

File: README.md [hunk 2]
```diff
@@ -1,59 +1,49 @@
...
-LegionCore development is completely open source; community involvement is highly encouraged.
-
-
-## Requirements
+* [Thank you](#thank-you)
 
-The software requirements for LegionCore are very similar to the one from TrinityCore.
-The main difference worth mentioning is that on Windows currently BOOST version 1.64 is required.
+# Introduction
+LegionCore is a **RPGMMO** framework for WOW Legion *(Build 26972)*. This core is based off of the UWOW core leak of 2020. Which was derived from an old version of [TrinityCore](https://github.com/TrinityCore/TrinityCore). LegionCore is completely opensource and is developed by the community. To submit a pull request please follow this template [here](submitting-fixes).
```
Comment (dufernst) [inline README.md:14]: Don't see a good reason to not stick to the usual abbreviation of MMORPG.

File: README.md [hunk 3]
```diff
@@ -1,59 +1,49 @@
...
 
-Software requirements are available in the [wiki](https://www.trinitycore.info/display/tc/Requirements) for
-Windows, Linux and macOS.
+If you want you can join the community discord: [here](https://discord.gg/uaP2aeJ7sj).
 
+# Requirements
+ 
+ ### Important
+ The main difference to note is that LegionCore requires Boost (1.64.0), Visual Studio 2017 & 2017 build tools, and MySQL (5.6.51).
```
Comment (dufernst) [inline README.md:21]: I would expect any visual studio 2017+ to work, are you sure e.g. 2019 does not work?
From what I remember from installing on Windows, I never had to manually download the build tools, those come with VS2017?

File: README.md [hunk 4]
```diff
@@ -1,59 +1,49 @@
...
+[Mac specific](https://www.trinitycore.info/en/install/requirements/macos)
 
-Detailed installation guides are available in the [wiki](https://www.trinitycore.info/display/tc/Installation+Guide) for
-Windows, Linux and macOS.
+# Install
+Most of the install steps are the same as the TrinityCore ones [here](https://www.trinitycore.info/en/install/Core-Installation).
 
-### MySQL 5.7+
+# Data Files
+To avoid any problems with World of Warcraft we will not be providing the data files for LegionCore instead you can aquire them from [here](https://www.emucoach.com/legion-7-3-5-/6945-repack-7-3-5-legion-wow-repack-wow-legion-7-3-5-repack-blizzlike-fun.html)
```
Comment (dufernst) [inline README.md:33]: Let's stick to:
`This core has been optimized for DBC/vmap/map/mmaps files from UWoW and are not provided as part of this source code package. Instead the data files for LegionCore can be acquired [here](https://www.emucoach.com/legion-7-3-5-/6945-repack-7-3-5-legion-wow-repack-wow-legion-7-3-5-repack-blizzlike-fun.html).`

File: README.md (hunk 2 above)
Comment (T-R-I-X) [reply README.md:14]: I mixed it up last night, I meant to say MMORPG I will fix that now

File: README.md (hunk 3 above)
Comment (T-R-I-X) [reply README.md:21]: From my experience on Windows 10 build 22621.1105 I had to use VS2017 and install the build tools manually.

Comment (T-R-I-X) [reply README.md:21]: It might've just been a one case thing?

Comment (dufernst) [reply README.md:21]: Also fine then, not too much to ask users anyway :)

File: README.md (hunk 1 above)
Comment (dufernst) [reply README.md:1]: Unless you are very attached to it, would you mind dropping the image from the change :)? Other than that looks good.

Comment (T-R-I-X) [reply README.md]: Dropped the image and changed it to a simple “LegionCore” header.

**Comment by dufernst** [comment]
@T-R-I-X if you are happy with the change as is I'll squash and merge it :)

**Comment by T-R-I-X** [comment]
@dufernst I will review my changes when I get home and make sure that’s everything that was needed to be included

File: README.md (hunk 3 above)
Comment (dufernst) [reply README.md:21]: @LoveBeforT @T-R-I-X lets change it to: Visual Studio 2017 or later then.

Comment (dufernst) [reply README.md:21]: After that I am fine with merging it.

Comment (T-R-I-X) [reply README.md]: You can merge now after reviewing one more time.

**Comment by dufernst** [comment]
Thanks again for the merge request, merged :)

**Pull request merged**